addons/l10n_latam_check/views/account_payment_view.xml

```diff
@@ -0,0 +1,197 @@
+<odoo>
+
+    <record model="ir.ui.view" id="view_account_payment_tree">
+        <field name="name">account.check.tree</field>
+        <field name="model">account.payment</field>
+        <field name="priority">20</field>
+        <field name="inherit_id" ref="account.view_account_payment_tree"/>
+        <field name="arch" type="xml">
+            <field name="name" position="attributes">
+                <attribute name="name">display_name</attribute>
+            </field>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_account_payment_search">
+        <field name="name">account.check.search</field>
+        <field name="model">account.payment</field>
+        <field name="priority">20</field>
+        <field name="inherit_id" ref="account.view_account_payment_search"/>
+        <field name="arch" type="xml">
+            <field name="journal_id" position="after">
+                <field name="l10n_latam_check_current_journal_id"/>
+            </field>
+            <filter name="date" position="after">
+                <separator/>
+                <filter string="Checks on hand" name="checks_on_hand" domain="[('state', '=', 'posted'), ('l10n_latam_check_current_journal_id.inbound_payment_method_line_ids.payment_method_id.code', '=', 'in_third_checks')]"/>
+            </filter>
+            <filter name="journal" position="after">
+                <filter name="groupby_third_check_current_journal" string="Third Check Current Journal" domain="[]" context="{'group_by': 'l10n_latam_check_current_journal_id'}"/>
+            </filter>
+        </field>
+    </record>
+
+    <record id="view_account_payment_form_inherited" model="ir.ui.view">
+        <field name="name">account.payment.form.inherited</field>
+        <field name="model">account.payment</field>
+        <field name="inherit_id" ref="account.view_account_payment_form" />
+        <field name="arch" type="xml">
+            <sheet position="before">
+                <div class="alert alert-warning mb-0" role="alert" attrs="{'invisible': [('l10n_latam_check_warning_msg', '=', False)]}">
+                    <field name="l10n_latam_check_warning_msg" nolabel="1"/>
+                </div>
+            </sheet>
+            <button name="print_checks" position="attributes">
```

Comment:
@jjscarafia 
remove mark sent funcionality (Ver si podemos borar botón, metodo y marcado automatico (also try to remove changing print_checks xml)